cypress/e2e/eureka/users/assigning-roles-user-edit-without-keycloak.cy.js

```diff
@@ -0,0 +1,168 @@
+import Users from '../../../support/fragments/users/users';
+import UsersCard from '../../../support/fragments/users/usersCard';
+import UserEdit from '../../../support/fragments/users/userEdit';
+import UsersSearchPane from '../../../support/fragments/users/usersSearchPane';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import AuthorizationRoles, {
+  SETTINGS_SUBSECTION_AUTH_ROLES,
+} from '../../../support/fragments/settings/authorization-roles/authorizationRoles';
+import TopMenu from '../../../support/fragments/topMenu';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+import { APPLICATION_NAMES } from '../../../support/constants';
+
+describe('Eureka', () => {
+  describe('Users', () => {
+    const randomPostfix = getRandomPostfix();
+    const testData = {
+      roleAName: `Auto Role A C499897 ${randomPostfix}`,
+      roleBName: `Auto Role B C499897 ${randomPostfix}`,
+      promotePath: '/users-keycloak/auth-users',
+      userBody: {
+        type: 'staff',
+        active: true,
+        username: `userc499897${randomPostfix}`,
+        patronGroup: '',
+        personal: {
+          lastName: `Last c499897 ${randomPostfix}`,
+          firstName: `First c499897 ${randomPostfix}`,
+          email: 'testuser@test.org',
+          preferredContactTypeId: '002',
+        },
+      },
+    };
+
+    const capabSetsToAssign = [
+      { type: 'Settings', resource: 'UI-Authorization-Roles Settings Admin', action: 'View' },
+      // { type: 'Data', resource: 'Capabilities', action: 'Manage' },
+      // { type: 'Data', resource: 'Role-Capability-Sets', action: 'Manage' },
```

Comment:
Should these lines of code be commented out?

Reply:
Forgot to remove. Will do that now. Thank you!